src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -247,9 +247,14 @@ internal abstract partial class AbstractInheritanceMarginService
                     foreach (var import in group)
                     {
                         var item = DefinitionItem.Create(
-                            [], [],
-                            new DocumentSpan(destinationDocument, import.DeclaringSyntaxReference!.Span),
-                            classifiedSpans: null);
+                            tags: [],
+                            displayParts: [],
+                            sourceSpans: [new DocumentSpan(destinationDocument, import.DeclaringSyntaxReference!.Span)],
+                            classifiedSpans: [null],
```

Comment:
we don't support just passing `default` fort his concept?  seems unfortunate to have to allocate.

Reply:
We don't. We expect sourceSpans length to match classifiedSpans length. We could support default.

Reply:
that would be my preference.  could you do that if you're already updating this pr? thanks!